RocketChat/Rocket.Chat.ReactNative
# Chore: Evaluate TextInput component

## Proposed changes
The proposal of this PR is unify our current components `containers/TextInput` and `presentation/TextInput` inside a single folder on `containers/TextInput`. 

After the evaluation, we decided rename the component `containers/TextInput` to `containers/TextInput/FormTextInput`. Bringing more sense to the responsibility from this file. 

## Issue(s)	


## How to test or reproduce
We should test all input on the app. E.g: 
* New workspace, login and register inputs; 
* test if the password input stills show/hide the text; 
* Search on the top of the app; 
* Change the status with a message; 

Go to settings -> language -> select the last one, then check if the input will start from right to left. 

## Screenshots

## Types of changes




## Checklist



## Further comments
